tests/e2e/Trustline.test.ts

```diff
@@ -117,8 +117,8 @@ describe('e2e', () => {
             setTimeout(() => {
               tl2.trustline.getUpdates(networkAddress).then(updates => {
                 const latestUpdate = updates[updates.length - 1]
-                expect(latestUpdate.address).to.equal(user1.address)
-                expect(latestUpdate.amount).to.equal(1500)
+                expect(latestUpdate.address.toLowerCase()).to.equal(user1.address.toLowerCase())
```

Comment:
Did that slip in by accident? There should be no lowercase needed anymore right?